plugins/CoreHome/angularjs/reporting-menu/reportingmenu.controller.js

```diff
@@ -218,7 +220,9 @@
         });
 
         $rootScope.$on('piwikPageChange', function (event) {
-            globalAjaxQueue.abort();
+            if (!initialLoad) {
```

Comment:
This should ensure that when the page loads initially it doesn't abort any ajax requests. Otherwise this makes problems when directly loading a popover with url params. Some UI tests where failing due to aborted requests.